challenges/advanced-variadic-generics/question.py

```diff
@@ -0,0 +1,25 @@
+"""
+TODO:
+
+Define a `Array` type that supports addition of arrays with the same shape.
+"""
+
+
+class Array:
+    def __add__(self, other):
+        ...
+
+    def add_dimension(self, x):
```

Comment:
It's unclear from test code what this should return

Reply:
Actually we might just remove `add_dimension`, I don't think it's necessary. wdyt?

Reply:
Ok, `add_dimension` method is used to demonstrate type concatenation. Let's remove it to keep the code simple.